minimal/server.go

```diff
@@ -239,14 +239,13 @@ func (j *jsonRPCHub) getState(root types.Hash, slot []byte) ([]byte, error) {
 }
 
 func (j *jsonRPCHub) GetAccount(root types.Hash, addr types.Address) (*state.Account, error) {
-	obj, err := j.getState(root, addr.Bytes())
-	if err != nil {
-		return nil, err
-	}
+	obj, _ := j.getState(root, addr.Bytes())
 	var account state.Account
+
 	if err := account.UnmarshalRlp(obj); err != nil {
 		return nil, err
 	}
```

Comment:
@Vuksan I've made wrong assumptions on how and what needs to be fixed in this case.

I have updated this PR's description accordingly.

Basically, I've just got rid of the `fmt.Errorf()` and replaced it with the simplest `errors.New()` because the severity of the slot & root not being found in the storage is determined from the caller funcs, so we shouldn't keep screaming `unable to get storage snapshot` when we are correctly returning the 0 for the next nonce or balance of an uninitialized address.

Reply:
@lazartravica I thought that the issue was that we are returning an error for the uninitialized account that we don't have in memory, instead of returning an account with zero balance, as geth does?

Reply:
@Vuksan 
We are  returning an error which in certain caller functions indicates that the account is uninitialized, allowing them to return some default values for uninitialized accounts such as having 0 balances or having the next nonce set to 0.

The only change is that the error is now created using `errors.New()`, and not `fmt.Errorf()` which will create an error and also print a message, because printing `error getting storage snapshot` when you're correctly getting back the balance of an uninitialized account as 0 is something we don't need.

You can take a look at the examples of how these errors returned from getState() are used to return gracefully even with an error here:
- https://github.com/0xPolygon/polygon-sdk/blob/97a06c1aa7a36758d7dae9efccf78935bfb620f7/jsonrpc/eth_endpoint.go#L438-L442
- https://github.com/0xPolygon/polygon-sdk/blob/97a06c1aa7a36758d7dae9efccf78935bfb620f7/jsonrpc/dispatcher.go#L396-L399

Reply:
@lazartravica `fmt.Errorf()` doesn't print a message AFAIK... it only formats a message - This is what got me confused. It would mean that the error is printed somewhere else

Reply:
@Vuksan you're absolutely right 🤦 

I probably got confused as I was unable to reproduce this exact issue with `error getting storage snapshot` when deploying the `pos-portal` smartcontracts with truffle, which was occurring 2 weeks ago.

I am going to revert these change.

Sorry for the mess, I'll squash & merge!

Reply:
And I was wondering what am I not getting 😄 
Cheers! 🍻